aula-pratica-03/ex1.c

```diff
@@ -0,0 +1,32 @@
+#include<stdio.h>
+
+int main()
+{
+    int vetor[20], index = 0, soma = 0, num;
+
+    printf("-----Parte A-----\n");
+    for(int i = 0; i < 20; i++) {
+        printf("Digite um numero: ");
+        scanf("%d", &num);
+        if(num % 2 == 0) {
+            vetor[index] = num;
+            soma += vetor[index];
+            index++;
+        }
+    }
+
+    printf("-----Parte B-----\n");
+    int maior = 0;
+    for(int i = 0; i < index; i++) {
+        printf("vetor[%d] = %d\n", i, vetor[i]);
+        if(vetor[i] > maior)
+            maior = vetor[i];
+    }
+    printf("Maior valor encontrado: %d\n", maior);
```

Comment:
also, if the user inputed only odd numbers this output would be wrong too, I'd put an if before giving this outputs like `if (input <= 0) { return 0; }`
```suggestion
    if (input <= 0) {
        return 0;
    }
    printf("Maior valor encontrado: %d\n", maior);
```